autoPlay.user.js

```diff
@@ -826,6 +825,12 @@ function goToLaneWithBestTarget() {
 			// Throw Money at Screen
 			enableAbilityItem(ITEMS.THROW_MONEY);
 		}
+		
+		if(!targetIsTreasureOrBoss && getGameLevel() >= 5000) {
```

Comment:
Pretty sure it doesn't need to be an extra `if`, and can just go into previous `else`